packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.stories.ts

```diff
@@ -23,7 +23,11 @@ type Story = StoryObj<typeof OnyxDataGrid>;
 
 export const Default = {
   args: {
-    columns: ["name", "age", "birthday"],
+    columns: [
+      { key: "name", label: "Firstname" },
```

Comment:
```suggestion
      { key: "name", label: "First name" },
```

Is it "Firstname" or "First name"?